js/packages/components/shared-components/BText.tsx

```diff
@@ -0,0 +1,22 @@
+import { useThemeColor } from '@berty-tech/store'
+import { useStyles } from '@berty-tech/styles'
+import React from 'react'
+import { Text, TextProps } from 'react-native'
+
+export const BText: React.FC<TextProps> = props => {
```

Comment:
I think you should rather choose between these two options to name this component:
1. `Text` so that you can't import both our component and react-native's one, the goal being I guess to use only ours starting now
2. if the first option is not desirable, choose a more descriptive name like `UnifiedText` or something like that, but not `BText` please 😉